src/buf/ext/mod.rs

```diff
@@ -124,6 +124,32 @@ pub trait BufMutExt: BufMut {
     fn writer(self) -> Writer<Self> where Self: Sized {
         writer::new(self)
     }
+
+    /// Creates an adaptor which will chain this buffer with another.
```

Comment:
Super nitpicky, but IIRC this spelling of "adaptor" is the British English spelling. Elsewhere in the docs, we use American English spellings, so I think it should be:
```suggestion
    /// Creates an adapter which will chain this buffer with another.
```